current_spring2026/pipeline.py

```
@@ -0,0 +1,231 @@
+
+    if (
+        GRAPH_RAG_ENABLED
+        and intent.query_type == "content_driven"
+        and intent.use_graph
+    ):
+        try:
+            expanded         = _expand_with_graph(intent, documents, top_k=GRAPH_TOP_K)
+            graph_docs_added = len(expanded) - len(documents)
+            documents        = expanded
+            graph_used       = graph_docs_added > 0
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)